src/sentry/static/sentry/less/group-detail.less

```diff
@@ -1735,7 +1735,8 @@ span.val {
     padding: 0 0 0 15px;
 
     .val-array-item {
-      display: block;
+      display: flex;
+      flex-direction: row;
```

Comment:
Why this change?

Reply:
This might have to do with some CSS changes from Dora

If it's `display: block`:
![image](https://user-images.githubusercontent.com/1748388/70595197-9a3e3080-1b97-11ea-960e-056b6a49fe55.png)

If it's `display: flex`:
![image](https://user-images.githubusercontent.com/1748388/70595198-9ca08a80-1b97-11ea-9c37-6c1b6afaa446.png)

I am testing this change on `staging` before merging.

Reply:
Looks good on `staging`.

Reply:
That's kind of weird as I haven't touched on any CSS. Anyway, if it looks good I'm good.